src/transformers/modeling_outputs.py

```diff
@@ -1751,3 +1751,38 @@ def logits(self):
             FutureWarning,
         )
         return self.reconstruction
+
+
+@dataclass
```

Comment:
Let's move this to the modeling file. At the moment we don't have other models using and it's highly specific to this one model

Reply:
Fixed in [this commit](https://github.com/huggingface/transformers/pull/28966/commits/0d2c3a11bc9c36557546633a69f6004c0948798e)